content/build/hydrobuild.md

```diff
@@ -54,58 +54,103 @@ Docker Desktop 4.23.0 and later versions ship with a Hydrobuild-compatible
 Buildx binary. Alternatively, you can download and install the binary manually
 from [this repository](https://github.com/docker/buildx-desktop).
 
-## Connecting to Hydrobuild
+## Connect to Hydrobuild
 
-To start building with Hydrobuild, you must create a new builder using the
-`docker buildx create` command. The builder is connected to Hydrobuild through
-an endpoint that you specify. The endpoint represents a single, isolated
-builder. Builder endpoints have the following format:
+To start using Hydrobuild, you must first add the builder's endpoint using the
+`docker buildx create` with the `cloud` driver.
 
-cloud://<org>/default
+{{< tabs group="ui" >}}
+{{< tab name="CLI" >}}
 
-`<org>` is the Docker organization that the builder is provisioned for.
+1. Sign in to your Docker account
 
-The builders have native support for the `linux/amd64` and `linux/arm64`
-architectures by default. This gives you a high-performance build cluster for
-building multi-platform images natively.
+   ```console
+   $ docker login
+   ```
 
-You can omit the `cloud://` protocol prefix from the endpoint when you create a
-builder using the `cloud` driver.
+2. Add the Hydrobuild endpoint.
 
-### Create a builder
+   ```console
+   $ docker buildx create --driver cloud <ORG>/default
```

Comment:
For November we're planning to also allow personal accounts so ORG might be confusing 🤔 I had the same problem when renaming things in the code and chose "account". That might not be ideal either so I'll throw in "namespace" as well 😃

Reply:
oh boy :) We had/have the same issue in Docker Scout. We eventually decided to keep using "org" there to keep it consistent with how we refer to this elsewhere. So I think we'll do the same here. But it would be nice to have a concept of "namespaces"...